go/cmd/sqlflowserver/e2e_common_cases.go

```diff
@@ -669,78 +685,94 @@ func caseXGBoostSparseKeyValueColumn(t *testing.T) {
 		return columns
 	}
 
-	dropDBSQL := ""
-	if testDBType == "hive" {
-		// Hive can only drop non-empty database in the CASCADE mode.
-		dropDBSQL = fmt.Sprintf("DROP DATABASE IF EXISTS %s CASCADE;", dbName)
-	} else {
-		dropDBSQL = fmt.Sprintf("DROP DATABASE IF EXISTS %s;", dbName)
+	if !isMaxCompute {
+		defer executeSQLFunc(dropDBSQL)
 	}
 
-	defer executeSQLFunc(dropDBSQL)
-
-	prepareDataSQL := []string{
-		dropDBSQL,
-		fmt.Sprintf(`CREATE DATABASE IF NOT EXISTS %s;`, dbName),
-		fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s.train (c1 VARCHAR(255), label_col FLOAT);`, dbName),
-		fmt.Sprintf(`INSERT INTO %s.train VALUES("1:0.5 3:-1.5 5:6.7", 0.5), ("3:-1.3 10:0.6 2:-3.4", -0.5);`, dbName),
+	var prepareDataSQL []string
+	if !isMaxCompute {
+		prepareDataSQL = []string{
```

Comment:
How about put the data preparation process in a function and call it in `sqlflowserver/testing.go:prepareTestData`?

Reply:
Done.